app/routes/sites/companies/company/edit.js

```diff
@@ -1,15 +1,22 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  model() {
+    return Ember.RSVP.hash({
+      company: this.modelFor('sites.companies.company'),
```

Comment:
how about assign this to a variable and use that variable. Little difference but looks better